Setup/deploy-shield.sh

```diff
@@ -27,11 +27,11 @@ if [ ! -f "$ES_YML_FILE" ]; then
     ES_YML_FILE=docker-compose_dev.yml
 fi
 
-function join_by() {
-    local IFS="$1"
-    shift
-    echo "$*"
-}
+
+if [ "$1" == "-no-browser" ]; then
```

Comment:
@BenyH 
This kind of use not allow me to use this flag with another flags together...
Add this flag to standard while loop